torch/csrc/Declarations.cwrap

```diff
@@ -2909,20 +2909,22 @@
     - function
   return: argument 0
   options:
-    - cname: add
+    - cname: add_scaled
       arguments:
         - arg: THTensor* result
           output: True
         - THTensor* self
         - real value
+        - arg: real alpha
+          default: AS_REAL(1)
     - cname: cadd
       aten_sparse: True
       arguments:
         - arg: THTensor* result
           output: True
         - arg: THTensor* self
           broadcast: other fallback
-        - arg: real value
```

Comment:
wouldn't it be nicer to call these args `scale`?

Reply:
Yeah, but they're the standard BLAS ops, where alpha appears in equations. Idk, might be just easier to remember. No strong preference